packages/create-workleap/package.json

```diff
@@ -14,7 +14,6 @@
   "dependencies": {
     "@clack/core": "^0.3.2",
     "@clack/prompts": "^0.6.3",
-    "@workleap/foundry": "workspace:*",
     "picocolors": "^1.0.0"
```

Comment:
Please use strict version for dependencies so we are in control of which version is installed and can deliberately bump version when needed.